scripts/lib/wic/canned-wks/nilrt-recovery-media.x64.wks

```diff
@@ -2,7 +2,7 @@
 # long-description: Creates an EFI and legacy bootable hybrid ISO image
 # which can be used on optical media as well as USB media.
 
-part /boot --source isoimage-isohybrid --sourceparams="esp_label=NIRECOVERY,esp_extra_blocks=5000,loader=grub-efi,image_name=nilrt-recovery-image.x64" --ondisk cd --label NIRECOVERY-CD
```

Comment:
Is there a client that consumes this image that needs to be notified?
In addition to that, you will also need to fix the build VM script as well as our ATS [configuration](https://codesearch.natinst.com/search/text?q=nilrt-recovery-image&fold_case=auto&regex=false&context=true).

Reply:
I have an ni-central PR which is fixing up the component references. I got the installers, but thanks for pointing out the ATS references. I'll switch those as well.